src/Geocoder/Provider/Chain.php

```diff
@@ -38,6 +40,9 @@ public function geocode($address)
     {
         $exceptions = [];
         foreach ($this->providers as $provider) {
+            if($provider instanceof LocaleAwareProvider && $this->getLocale() !== null){
```

Comment:
Could you please use the coding standards of this project?

* 1 space before `(`
* `null !== ...` rather than the other way around

Does it make sense to check whether locale is null? It does not really matter IMO.